scope

```diff
@@ -90,8 +90,14 @@ check_not_running() {
 }
 
 launch_command() {
+    local args="-v /etc/weave:/etc/weave"
```

Comment:
Why is this needed? It makes the container stateful and breaks reproducibility.  

For instance, I spent 30 mins going in circles, because it makes the app targets get appended between scope launches. See https://github.com/weaveworks/scope/blob/master/docker/entrypoint.sh#L168

```
echo "$MANUAL_APPS" >>/etc/weave/apps
```

That should had probably just had been `"$MANUAL_APPS" >/etc/weave/apps` from the beginning but it points out why it's a bad idea to use local volumes.